src/components/media-image.js

```diff
@@ -162,7 +150,6 @@ AFRAME.registerComponent("media-image", {
           break;
         default:
           this.mesh.material.transparent =
-            !this.data.batch ||
```

Comment:
Here, I noticed that `this.mesh.material.transparent` should be alwayt `true` because `data.batch` has been removed (`false`). I'm making a follow up PR now.